src/Tribe/Log/Service_Provider.php

```diff
@@ -145,4 +147,30 @@ public function add_logging_engine( array $logging_engines = [] ) {
 
 		return $logging_engines;
 	}
+
+	/**
+	 * Enables logging in the service provider, if not already enabled.
+	 *
+	 * @since TBD
+	 */
+	public function enable(  ) {
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: Expected exactly one space before closing parenthesis; "  " found.
* :no_entry_sign: Expected 0 spaces between parenthesis of function declaration; 2 found
* :no_entry_sign: Opening brace should be on a new line

<sub>hash: c51cb8e1ef387e1bc7dbbfa748fdd6af</sub>